test/unit/logic/block.js

```diff
@@ -317,40 +326,65 @@ describe('block', () => {
 		done();
 	});
 
-	describe('with valid block and data', () => {
-		beforeEach(done => {
-			data = _.cloneDeep(validDataForBlock);
-			transactions = _.values(transactionsByTypes);
-			done();
+	beforeEach(done => {
+		data = _.cloneDeep(validDataForBlock);
+		transactions = _.values(transactionsByTypes);
+		done();
+	});
+
+	describe('create', () => {
+		var blockNormalizeStub;
+
+		before(() => {
+			blockNormalizeStub = sinonSandbox
+				.stub(block, 'objectNormalize')
+				.returnsArg(0);
+
+			transactionStub.getBytes.returns(Buffer.from('dummy transaction bytes'));
+			return transactionStub.objectNormalize.returnsArg(0);
 		});
 
-		describe('create', () => {
-			var blockNormalizeStub;
+		after(() => {
+			blockNormalizeStub.reset();
+			transactionStub.getBytes.reset();
+			return transactionStub.objectNormalize.reset();
+		});
 
-			before(() => {
-				blockNormalizeStub = sinonSandbox
-					.stub(block, 'objectNormalize')
-					.returnsArg(0);
+		describe('when one of all transaction types are present', () => {
```

Comment:
`when each of all supported...`

Reply:
I think the existing comment make more relevant, what do you think, should we change?

Reply:
because we're testing block that contains all transaction types - one transaction of each supported type, `one of all` means one transaction type of all supported types (so assume that only one transaction is present)